guice/src/main/java/com/flipkart/gjex/grpc/interceptor/FilterInterceptor.java

```diff
@@ -169,9 +199,23 @@ public void onCancel() {
                 Context previous = attachContext(contextWithHeaders);   // attaching headers to gRPC context
                 try {
                     super.onCancel();
-                } catch (RuntimeException ex) {
-                    handleException(call, ex);
-                    grpcFilters.forEach(filter -> filter.doHandleException(ex));
+                    grpcFilters.forEach(filter -> filter.doHandleException(new Exception()));
```

Comment:
check how to set more descriptive exception containing grpc status code here, like DEADLINE_EXCEEDED